deployment/terraform/modules/vpc/main.tf

```diff
@@ -0,0 +1,99 @@
+resource "aws_vpc" "vpc" {
+  cidr_block           = "${var.vpc_cidr_block}"
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+
+  tags {
+    Name = "${terraform.workspace} (terraform)"
+  }
+}
+
+resource "aws_subnet" "public_subnet" {
+  vpc_id                  = "${aws_vpc.vpc.id}"
+  cidr_block              = "${var.public_cidr_block}"
+  availability_zone       = "${var.availability_zone_a}"
+  map_public_ip_on_launch = true
+
+  tags {
+    Name = "${terraform.workspace} (terraform public subnet)"
+  }
+}
+
+resource "aws_subnet" "private_subnet_a" {
+  vpc_id            = "${aws_vpc.vpc.id}"
+  cidr_block        = "${var.private_cidr_block_a}"
+  availability_zone = "${var.availability_zone_a}"
+
+  tags {
+    Name = "${terraform.workspace} (terraform private subnet a)"
+  }
+}
+
+// This subnet currently isn't utilized other than to satisfy the
```

Comment:
Single line comments start with #
Multi-line comments are wrapped with /* and */
https://www.terraform.io/docs/configuration/syntax.html#terraform-syntax

Reply:
will fix